.github/workflows/pr.yml

```diff
@@ -15,8 +15,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - os: ubuntu-latest
+          - os: windows-latest
+          - os: macos-latest
+            target: x86_64
+          - os: macos-14
+            target: arm64
     runs-on: ${{ matrix.os }}
+    env:
+      TARGET: ${{ matrix.target }}
```

Comment:
For the Dissolve overhaul I no longer needed to pass the arch to the build stage like this - assuming native arch on the runner is enough.